furusiya/model/weapons/bow.py

```diff
@@ -0,0 +1,6 @@
+class Bow:
```

Comment:
Docstring please. Should mention: Bow has a plain melee attack, and is used to tell if we can fire arrows.